data/ggjGameStory.xml

```diff
@@ -3,27 +3,103 @@
 		<storyNode id="holySymbol1" type="satellite" lastNode="false">
          	<teaserText>The town blacksmith comes to you with an interesting proposal...</teaserText>
          	<eventText> The blacksmith stands before you, twisting his beard in excitement. He's hoping that you can fund the creation of ornate metal-worked holy symbols for your burgeoning 	faith. </eventText>
-         <functionalDescription>
-            <tag id="wealthTheme"/>
-	         </functionalDescription>
-	         <prerequisite>
+         	<functionalDescription>
+				<prominence id="dawgCharacter">3</prominence>	        
```

Comment:
Now you just need to update this for this story. ;) You'll probably want to use just 1 for prominence values for now.